mw-webapp/src/logic/waysTable/MentoringWaysTable.tsx

```diff
@@ -0,0 +1,42 @@
+import {useEffect, useState} from "react";
+import {WayPreviewDAL} from "src/dataAccessLogic/WayPreviewDAL";
+import {columns} from "src/logic/waysTable/columns";
+import {WaysTable} from "src/logic/waysTable/WaysTable";
+import {WayPreview} from "src/model/businessModelPreview/WayPreview";
+
+/**
+ * Mentoring ways table props
+ */
+interface MentoringWaysTableProps {
+
+  /**
+   * User Uuid
+   */
+  uuid: string;
+}
+
+/**
+ * Render table of mentoring ways preview
+ */
+export const MentoringWaysTable = (props: MentoringWaysTableProps) => {
+  const [ownWays, setOwnWays] = useState<WayPreview[]>([]);
+
```

Comment:
If you load mentoringWays why state calls ownWays

Reply:
Already renamed